core/src/builder.ts

```diff
@@ -160,7 +160,12 @@ function Builder<Model>({
                   : type === 'rest' || propToSet === 'buildRest'
                     ? 'rest'
                     : 'default';
-              transformed = (transformers?.[transformersType]?.transform({
+
+              // If we don't find a transformer, use REST by default as this only
+              // happens in the compatibility mode, where we don't have a 'default' transformer.
+              const transformer =
```

Comment:
This changes allows for some edge cases with compatibility between new and legacy data models.